saga/init-customer.sql

```diff
@@ -0,0 +1,2 @@
+create schema customer;
```

Comment:
I assume these 3 .sql files to be there just in case someone wants do use them for init purposes? the actual DB schema is created by ORM setting in the `application.properties` right?

Reply:
No, these files actually are executed (see Docker Compose set-up, they are copied to init.d). The reason being that ORM doesn't _create_ the schema itself, it only _populates_ it with all the required tables, sequences etc.